Tests/PackageGraphTests/PubgrubTests.swift

```diff
@@ -511,6 +512,27 @@ final class PubgrubTests: XCTestCase {
         ])
     }
 
+    func testUpdatePackageIdentifierAfterResolution() {
+        let fooRef = PackageReference(identity: "foo", path: "https://some.url/FooBar")
+        let foo = MockContainer(name: fooRef, dependenciesByVersion: [v1: []])
+        foo.manifestName = "foobar"
```

Comment:
Can you set this to anything other than "foo" or "foobar" to avoid the test from passing when it should have failed due to some case insensitive check?